### DISCUSSION TITLE
Librum Web client

### DISCUSSION THREAD

**Discussion by tbleiker** [Librum-Reader/Librum#132] [General]
Do you consider a web client as well? 

For reading, I certainly prefer an app. But time and again, I'm glad that I can quickly access my books online (I currently use calibre/calibre-web and bookfusion for books which I highlight).

**Comment by DavidLazarescu** [comment]
We are considering a web client, but the native applications will be the focus for now. Once they work well, we will look into the web app.

**Comment by Raiper34** [comment]
I would like to have web interface too :)
